xmtp_mls/src/groups/mod.rs

```diff
@@ -1074,7 +1076,11 @@ pub fn build_extensions_for_permissions_update(
         PermissionUpdateType::UpdateMetadata => {
             let mut metadata_policy = existing_policy_set.update_metadata_policy.clone();
             metadata_policy.insert(
-                update_permissions_intent.metadata_field_name.unwrap(),
```

Comment:
this one seems like we could make a default, like `unwrap_or("DEFAULT METADATA FIELD NAME")` instead of an error